wifi-connect-exclave/Dockerfile.template

```diff
@@ -51,16 +52,16 @@ ENV PATH /root/.cargo/bin:${PATH}
 WORKDIR /usr/src/exclave
 
 # clone exclave
-RUN git clone https://github.com/exclave/exclave . && git checkout ${EXCLAVE_VERSION}
-
-# build exclave
-RUN cargo build
+RUN git clone https://github.com/exclave/exclave . && \
```

Comment:
@klutchell Saving layers like this would help right?

Reply:
correct! we could combine even more RUN commands but then it becomes an unreadable mess and doesn't benefit from caching unchanged layers so builds take longer